packages/web3-eth/src/web3_eth.ts

```diff
@@ -291,6 +1100,14 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 
 	// TODO Decide what to do with transaction.to
 	// https://github.com/ChainSafe/web3.js/pull/4525#issuecomment-982330076
+	/**
+	 * Executes a message call transaction, which is directly executed in the VM of the node, but never mined into the blockchain.
```

Comment:
Lets be more descriptive for this function
```suggestion
	 * Executes a message call in EVM without creating transaction. It does not publish anything to the blockchain and also does not consume any gas.
```

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5251/commits/c54e3f6d239196516566a7b5c823e344c9727dd2) commit